Makefile

```diff
@@ -42,18 +44,57 @@ deps:
 	# 	aws s3 mb s3://rig.${OWNER}.${PROJECT}.${REGION}.build-support.${ENV} --region "${REGION}" # Build artifacts, etc
 	# @aws s3api put-bucket-versioning --bucket "rig.${OWNER}.${PROJECT}.${REGION}.build-support.${ENV}" --versioning-configuration Status=Enabled --region "${REGION}"
 
-# Destroy dependency S3 buckets, only destroy if empty
-delete-deps:
+delete-foundation-deps:
+	@scripts/empty-s3-bucket.sh rig.${OWNER}.${PROJECT}.${REGION}.foundation.${ENV}
 	@aws s3 rb --force s3://rig.${OWNER}.${PROJECT}.${REGION}.foundation.${ENV}
+	@scripts/empty-s3-bucket.sh rig.${OWNER}.${PROJECT}.${REGION}.compute-ecs.${ENV}
 	@aws s3 rb --force s3://rig.${OWNER}.${PROJECT}.${REGION}.compute-ecs.${ENV}
+	@scripts/empty-s3-bucket.sh rig.${OWNER}.${PROJECT}.${REGION}.app.${ENV}
 	@aws s3 rb --force s3://rig.${OWNER}.${PROJECT}.${REGION}.app.${ENV}
 	# @aws s3 rb --force s3://rig.${OWNER}.${PROJECT}.${REGION}.build-support.${ENV}
+	@scripts/empty-s3-bucket.sh rig.${OWNER}.${PROJECT}.${REGION}.build
 	@aws s3 rb --force s3://rig.${OWNER}.${PROJECT}.${REGION}.build
 
+create-deps:
+	@echo "Create SSM build parameters: /${OWNER}/${PROJECT}/build"
+	@read -p 'GitHub OAuth Token: ' REPO_TOKEN; \
+		aws cloudformation create-stack --stack-name "${OWNER}-${PROJECT}-deps" \
+                --region ${REGION} \
+			--template-body "file://cloudformation/deps/deps.yaml" \
+			--disable-rollback \
+			--capabilities CAPABILITY_NAMED_IAM \
+			--parameters \
+				"ParameterKey=ParameterStoreNamespace,ParameterValue=/${OWNER}/${PROJECT}" \
+				"ParameterKey=RepoToken,ParameterValue=${REPO_TOKEN}" \
+			--tags \
+				"Key=Owner,Value=${OWNER}" \
+				"Key=Project,Value=${PROJECT}"
+	@aws cloudformation wait stack-create-complete --stack-name "${OWNER}-${PROJECT}-deps" --region ${REGION}
+
+update-deps:
+	@echo "Update SSM build parameters: /${OWNER}/${PROJECT}/build"
+	@read -p 'GitHub OAuth Token: ' REPO_TOKEN; \
+		aws cloudformation update-stack --stack-name "${OWNER}-${PROJECT}-deps" \
+                --region ${REGION} \
+			--template-body "file://cloudformation/deps/deps.yaml" \
+			--capabilities CAPABILITY_NAMED_IAM \
+			--parameters \
+				"ParameterKey=ParameterStoreNamespace,ParameterValue=/aochsner/bookit" \
```

Comment:
This should be changed to the owner as referenced below?

Reply:
Yep absolutely.  Thanks!